requirements.txt

```diff
@@ -1,5 +1,5 @@
 fire >= 0.4
-fsspec >= 2021.7.0
+fsspec >= 2021.7.0, < 2024.3.0
```

Comment:
🟢 slight preference to comment with a link to this PR to save future editors a blame lookup.

Reply:
Done in https://github.com/related-sciences/articat/pull/59/commits/892ad57a582283d488d0f22d3670205e29ee0136